src/components/overview-header/overview-header.js

```diff
@@ -17,15 +17,18 @@ class OverviewHeader extends React.PureComponent {
       <a href={props.changelogLink}>View changelog</a>
     );
 
-    if (!versionEl && !changelogLinkEl) {
+    const tag = props.tag && this.buildTag();
+
+    if (!versionEl && !changelogLinkEl && !tag) {
       return null;
     }
 
     return (
-      <p>
+      <div className="mb12">
+        {tag}
```

Comment:
I don't really like how the placement of the tag looks (I think it kind of gets lost and doesn't really align with the list elements), but I'm not sure if it's just because I'm used to it being next to the name of the product... 

<img width="1412" alt="Screen Shot 2020-11-05 at 3 01 39 PM" src="https://user-images.githubusercontent.com/10479155/98296059-fb24e300-1f77-11eb-83e2-8361da65943a.png">

Reply:
That's a fair point. I'll put it back to where it was!